tests/integration-test/src/tests/utils.rs

```diff
@@ -131,21 +156,34 @@ pub fn add_tc_pass(
     priority: u32,
     globals: Option<Vec<&str>>,
     proceed_on: Option<Vec<&str>>,
+    load_type: &LoadType,
 ) -> Result<String> {
     let p = priority.to_string();
 
-    let mut args = vec!["load-from-image"];
+    let mut args = Vec::new();
+
+    match load_type {
+        LoadType::Image => {
+            args.extend(["load-from-image"]);
```

Comment:
nit: Why `.extend` and not `.push`?

Reply:
Good point. It had more items before the change.  I meant to change it to a push, but forgot to go back and do it.  I'll make the change.

Reply:
I guess I also was making it consistent everywhere.  I'm changing it to push everywhere I'm just adding a single element.